orcestra_book/data_policy.md

```diff
@@ -0,0 +1,11 @@
+# Data Policy
+
+ORCESTRA encourages free use and access to data without strict control. We trust in the integrity of the scientific community to honor the contributions of those who collected these data.
```

Comment:
```suggestion
ORCESTRA encourages free use and access to data ~~without strict control~~. We trust in the integrity of the scientific community to honor the contributions of those who collected these data.
```

Reply:
I agree, the words are more or less redundant. Removing them simplifies the language which is good.